vllm/worker/model_runner_base.py

```diff
@@ -218,6 +224,20 @@ def prepare_model_input(
         """
         raise NotImplementedError
 
+    def prepare_model_input_align_worker(
```

Comment:
is that necessary to add a base_method here? Seems it is only used in hpu_model_runner.py?